packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerCalendarHeaderCellStyles.ts

```diff
@@ -9,7 +9,19 @@ export const datepickerCalendarHeaderCellStyles: ComponentSlotStylesPrepared<
   root: ({ variables: v }): ICSSInJSStyle => {
     return {
       textAlign: 'center',
+      verticalAlign: 'middle',
+
       fontWeight: v.calendarHeaderCellFontWeight,
+
+      height: v.calendarHeaderCellHeight,
+      width: v.calendarHeaderCellWidth,
+
+      paddingTop: v.calendarHeaderCellPaddingTop,
+      paddingBottom: v.calendarHeaderCellPaddingBottom,
+      paddingLeft: v.calendarHeaderCellPaddingLeft,
+      paddingRight: v.calendarHeaderCellPaddingRight,
```

Comment:
```suggestion
      padding: v.calendarHeaderCellPadding,
```

Let's have this as a single padding